docs/docs/administration/reverse-proxy.md

```diff
@@ -15,7 +15,34 @@ While the reverse proxy provided by Immich works well for basic deployments, som
 
 ## Adding a Custom Reverse Proxy
 
-Users can deploy a custom reverse proxy that forwards requests to Immich's reverse proxy. This way, the new reverse proxy can handle TLS termination, load balancing, or other advanced features, while still delegating routing decisions to Immich's reverse proxy. All reverse proxies between Immich and the user must forward all headers and set the `Host`, `X-Forwarded-Host`, `X-Forwarded-Proto` and `X-Forwarded-For` headers to their appropriate values. By following these practices, you ensure that all custom reverse proxies are fully compatible with Immich.
+Users can deploy a custom reverse proxy that forwards requests to Immich's reverse proxy. This way, the new reverse proxy can handle TLS termination, load balancing, or other advanced features, while still delegating routing decisions to Immich's reverse proxy. All reverse proxies between Immich and the user must forward all headers and set the `Host`, `X-Forwarded-Host`, `X-Forwarded-Proto` and `X-Forwarded-For` headers to their appropriate values. Additionally, your reverse proxy should allow for big enough uploads. By following these practices, you ensure that all custom reverse proxies are fully compatible with Immich.
+
+### Nginx example config
+
+Below is an example config for nginx:
+
+```nginx
+server {
+    server_name <snip>
+
+    # https://github.com/immich-app/immich/blob/main/nginx/templates/default.conf.template#L28
+    client_max_body_size 50000M;
+
+    location / {
+        proxy_pass http://<snip>/2283;
```

Comment:
This should be `http://<snip>:2283`

Reply:
hello @fwsmit Can you help address the issue above?